docs/quick-start.md

```diff
@@ -3,22 +3,374 @@ title: Getting Started
-1. [Provision a MySQL database](quick-start-gcp.md) on GCP. 
+1. [Install Crossplane and the GCP stack](#install-crossplane).
+1. [Configure Crossplane to connect to GCP](#configure-crossplane-to-connect-to-gcp).
+1. [Provision a MySQL instance](#provision-a-mysql-instance).
+1. [Enable Dynamic Provisioning of MySQL databases](#enable-dynamic-provisioning).
+
+## Install Crossplane
+
+We'll start by installing Crossplane using [Helm]. You'll need a working
+Kubernetes cluster ([minikube] will do just fine). Crossplane is currently in
+alpha, so we'll use the `alpha` channel:
+
+```bash
+# Crossplane lives in the crossplane-system namespace by convention.
+kubectl create namespace crossplane-system
+
+helm repo add crossplane-alpha https://charts.crossplane.io/alpha
+helm install --name crossplane --namespace crossplane-system crossplane-alpha/crossplane
+```
+
+Once Crossplane is installed we'll need to install the GCP [stack]. Installing
+the GCP stack teaches Crossplane how to provision and manage things in GCP. You
+install it by creating a `ClusterStackInstall`:
+
+```yaml
+apiVersion: stacks.crossplane.io/v1alpha1
+kind: ClusterStackInstall
+metadata:
+  name: stack-gcp
+  namespace: crossplane-system
+spec:
+  package: "crossplane/stack-gcp:master"
+```
+
+Save the above as `stack.yaml`, and apply it by running:
+
+```bash
+kubectl apply -f stack.yaml
+```
+
+We've now installed Crossplane with GCP support! Take a look at the guide to
+[installing Crossplane] for more installation options, and to learn how to
+install support for other cloud providers such as Amazon Web Services and
+Microsoft Azure.
+
+## Configure Crossplane to Connect to GCP
+
+We've taught Crossplane how to work with GCP - now we must tell it how to
+connect to your GCP project. We'll do this by creating a Crossplane `Provider`
+that specifies the project name and some GCP service account credentials to use:
+
+```yaml
+apiVersion: gcp.crossplane.io/v1alpha2
+kind: Provider
+metadata:
+  name: example-provider
+spec:
+  # Make sure to update your project's name here.
+  projectID: my-cool-gcp-project
+  credentialsSecretRef:
+    name: example-gcp-credentials
+    namespace: crossplane-system
+    key: credentials.json
+```
+
+Save the above `Provider` as `provider.yaml`, then run:
+
+```bash
+kubectl -n crossplane-system create secret example-gcp-credentials --from-file=credentials.json
+kubectl apply -f provider.yaml
+```
+
+Crossplane can now manage your GCP project! Your service account will need the
+CloudSQL Admin role for this guide. Check out GCP's [Getting Started With
+Authentication] guide if you need help creating a service account and
+downloading its `credentials.json` file, and Crossplane's [GCP provider
+documentation] for detailed instructions on setting up your project and service
+account permissions.
+
+## Provision a MySQL Instance
+
+GCP provides MySQL databases using [CloudSQL] instances. Crossplane uses a
```

Comment:
nit: GCP provisions

Reply:
I meant to write provides. Provisions means more like "GCP creates MySQL databases", while I meant literally "GCP provides you with MySQL databases via CloudSQL", which is also true.